website/frontend/src/pages/ExploreData/index.js

```diff
@@ -441,115 +512,152 @@ export const ExploreOrganisationRegistry = () => {
+          </div>
+          <div className="content">
+            <h2>Visualise air quality information.</h2>
+            <p>
+              Access real-time and historic air quality information across Africa through our
+              easy-to-use air quality analytics dashboard or mobile app.
+            </p>
+          </div>
+        </div>
+        <div className="right-section">
+          <div className="nav-icon">
+            <Link to="/">
+              <CloseIcon />
+            </Link>
+          </div>
+          <div className="nav">
+            <div className="nav-option">
+              <div className="img-wrapper">
+                <div className="img-1">
+                  <MobileApp />
+                </div>
+              </div>
+              <h6>Discover the quality of air you are breathing.</h6>
+              <Link to="/explore-data/download-apps">
+                <button className="nav-button">Download App</button>
+              </Link>
+            </div>
+            <div className="nav-option">
+              <div className="img-wrapper">
+                <div className="img-2">
+                  <AirqualityPlatform />
                 </div>
-                <div className="right-section">
-                    <div className="nav-icon"><Link to="/"><CloseIcon /></Link></div>
-                    <div className="nav">
-                        <div className="nav-option">
-                            <div className="img-wrapper"><div className="img-1"><MobileApp /></div></div>
-                            <h6>Discover the quality of air you are breathing.</h6>
-                            <Link to="/explore-data/download-apps"><button className="nav-button">Download App</button></Link>
-                        </div>
-                        <div className="nav-option">
-                            <div className="img-wrapper">
-                                <div className="img-2"><AirqualityPlatform /></div>
-                            </div>
-                            <h6>An interactive air quality analytics platform</h6>
-                            <Link to="/explore-data/get-started"><button className="nav-button">Air Quality Dashboard</button></Link>
-                        </div>
-                    </div>
+              </div>
+              <h6>An interactive air quality analytics platform</h6>
+              <Link to="/explore-data/get-started">
+                <button className="nav-button">Air Quality Dashboard</button>
+              </Link>
+            </div>
+            <div className="nav-option">
+              <div className="img-wrapper">
+                <div className="img-1">
+                  <AirqualityPlatform />
                 </div>
+              </div>
+              <h6>Learn how to access air quality data.</h6>
+              <button className="nav-button">
+                <a href={UserManual} download="AirQo Analytics Platform User Manual">
```

Comment:
Could we utilize `target="_blank"` for this part? Just to open the pdf document in another tab? Just to ensure that our users can easily return to our application after viewing the PDF?

For security purposes, we could also add the `rel="noopener noreferrer"` attribute to prevent the new tab from having access to the "parent" page

Reply:
@Baalmart This is a download file, not a view file. So no need to open a link

Reply:
thanks  @FaithDaka